Makefile

```diff
@@ -0,0 +1,13 @@
+init:
+	pipenv install -d
+
+test:
+	pipenv run pytest tests
+
+coverage:
+	pipenv run pytest --cov=redfish_client --cov-report=html tests
+	xdg-open htmlcov/index.html
+
+publish:
+	python3 setup.py sdist bdist_wheel
+	python3 -m twine upload dist/*
```

Comment:
In a followup PR we'd probably configure CircleCI to push automatically when pushed to a branch like `release/0.2.0` right? Anyway, good job providing the makefile to reduce human memory consumption 👍

Reply:
Automating PyPI package creation is an ultimate goal, but since we have not used CircleCI for this previously (we used Travis up until now for all of our projects), someone needs to read the docs about handling secrets with CircleCI before we enable this.